apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -908,7 +908,7 @@ export class RushConfiguration {
     this._yarnCacheFolder = path.resolve(path.join(this._commonTempFolder, 'yarn-cache'));
```

Comment:
`rush purge` has special logic to avoid deleting the folder that it's running out of.  Make sure it still works with this change